cgr/cgr.py

```diff
@@ -57,12 +57,15 @@ async def update_ratings_whosaidit(self, ctx, players, winners):
         elif winners:
             players.remove(winners[0])
             loser_rating = await self.get_average_rating(players, "whosaidit")
-            winner_rating = await self.get_average_rating(winners, "whosaidit")
+            winner_rating = await self.get_average_rating([winners[0]], "whosaidit")
             r2 = math.pow(10, loser_rating/400)
-            await self.update_ratings_helper(ctx, winners, r2, 1, "whosaidit")
+            await self.update_ratings_helper(ctx, winners, r2, 1.0 + (0.2 * len(players)), "whosaidit")
             r2 = math.pow(10, winner_rating/400)
             for player in players:
```

Comment:
`update_ratings_helper` takes a `team` parameter that already iterates over each member. Can't it be:

```suggestion
            self.update_ratings_helper(ctx, [winners], r2, 0, "whosaidit")
            self.update_ratings_helper(ctx, [p for p in players not in winners], r2, 0.4, "whosaidit")
```